src/main/java/controllers/SwingBoard.java

```diff
@@ -1,36 +1,54 @@
 package controllers;
 
 import common.Outcomes;
+import common.Winner;
 import entities.ComputerShip;
+import entities.UserShip;
 import service.GameState;
 import entities.Guess;
 
 import javax.swing.JPanel;
 import javax.swing.JFrame;
+import java.awt.Component;
+import java.awt.Cursor;
 import javax.swing.JTextField;
-import javax.swing.BorderFactory;
 import javax.swing.JOptionPane;
-
+import javax.swing.SwingConstants;
 import java.awt.Graphics;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import javax.swing.BorderFactory;
+
 import java.awt.Dimension;
 import java.awt.GridLayout;
 import java.awt.BorderLayout;
 import java.awt.Color;
-import java.awt.event.MouseEvent;
-import java.awt.event.MouseListener;
-
 import java.util.logging.Logger;
+import java.util.ArrayList;
+import java.util.Iterator;
 
 
 public class SwingBoard extends JPanel implements MouseListener {
     private static final int PANEL_LENGTH = 10;
     private static final int PANEL_WIDTH = 10;
+    private static final int MAX_USER_SHIPS = 6;
 
     private Logger log = Logger.getLogger("Panel Location");
     private PlayerSelection playerSelection;
     private GameState state;
+    private ArrayList list;
     private JPanel mainPanel, footerPanel;
     private JFrame frame;
+    private ArrayList<Panel> listOfPanels = new ArrayList<Panel>();
+    private int computerPanel;
+    private int computerPanelClicked = 0;
+    private int shipCreator = 0;
+    private ArrayList<Integer> intValueOfPanel = new ArrayList<>();
+    private int userShips = 0;
+    private boolean setUpMode = false;
```

Comment:
Is this used? Check other variables are also used - your IDE should point out any redundant code